src/index.js

```diff
@@ -58,11 +69,12 @@ export default function svgSprite(options = {}) {
       }
 
       const filename = path.basename(id, '.svg')
+      const { content, viewBox, id: symbolId } = createSymbol(code, filename)
 
-      convertedSvgs.set(id, createSymbol(code, filename))
+      convertedSvgs.set(id, content)
 
       return {
-        code: ''
+        code: `export default {id: '${symbolId}_usage', viewBox: '${viewBox}', url: '${publicPath}${spriteFilename}#${symbolId}', toString() { return this.url },}`
```

Comment:
I haven't used webpack svg loader, can you explain me, why we need to add `_usage` to symbol `id` here and what are `url` and `toString()` used for?